app/assets/stylesheets/application.scss

```diff
@@ -118,36 +118,6 @@ h1, h2, h3, h4, h5, h6 {
   color: var(--Secondary-Red-Dark, #b00002);
 }
 
-/* Used for the delete button when adding users */
-.transparent-button {
-  background-color: transparent;
-  background-repeat: no-repeat;
-  border: none;
-  cursor: pointer;
-  overflow: hidden;
-  outline: none;
-}
-
-/* Data users in the Edit form */
-#data-users-table {
-  border-color: gray;
-  border-style: solid;
-  border-width: 1px;
-}
-
-#data-users-table > thead {
-  background-color: lightgray;
-}
-
-.added-user-textbox {
-  border: none !important;
-  background-color: white !important;
-}
-
-.delete-user-row {
-  color: red;
-}
-
```

Comment:
These styles were either replaced or moved to `app/assets/stylesheets/_project.scss`